redirects.json

```diff
@@ -4,6 +4,11 @@
   { "from": "/archive/(.*)$", "to": "/", "permanent": false },
   { "from": "/our-sponsors", "to": "/sponsors", "permanent": true },
   { "from": "/about-freecodecamp", "to": "/about", "permanent": true },
+  {
+    "from": "/sql-order-by-statement-example-sytax",
+    "to": "/sql-order-by-statement-example-syntax",
```

Comment:
Hey Kris,
At what point does this go through? That is, the `to` address is still a 404.

Reply:
Sorry for the late response @ShaunSHamilton, just seeing this now.

Once this is merged, I'll go ahead and change the slug on the live article from `sql-order-by-statement-example-sytax` to `/sql-order-by-statement-example-syntax`. Then I can deploy the new redirects.

This should minimize any downtime.